presto-hive/src/main/java/io/prestosql/plugin/hive/rubix/RubixModule.java

```diff
@@ -22,6 +22,7 @@
 import com.qubole.rubix.prestosql.CachingPrestoDistributedFileSystem;
```

Comment:
Update commit message to:
```
Do not allow impersonation with DefaultRubixHdfsInitializer
```

Reply:
The check was actually moved from `io.prestosql.plugin.hive.authentication.HiveAuthenticationModule#setup`. updated code